test-lib-openshift.sh

```diff
@@ -322,8 +322,19 @@ function ct_os_cluster_up() {
   fi
 
   mkdir -p ${dir}/{config,data,pv}
-  oc cluster up --host-data-dir=${dir}/data --host-config-dir=${dir}/config \
-                --host-pv-dir=${dir}/pv --use-existing-config --public-hostname=${cluster_ip}
+  case $(oc version| head -n 1) in
+    "oc v3.1"*)
```

Comment:
This will match `v3.1` as well as `v3.10`.  Is that OK?

Reply:
Well, kinda, I did not care about 3.1 since it is dead.. But I guess I can change it to `3.1?.*)` which should match what was supposed to match.